.github/workflows/ci-pipeline.yml

```diff
@@ -117,7 +117,7 @@ jobs:
             verification_mode="$(cat $config_file | grep 'verification-mode' | cut -d'=' -f2)"
           fi
           echo "verification_mode=${verification_mode}" >> $GITHUB_OUTPUT
-      - uses: vmware-labs/vmware-image-builder-action@0.4.10
+      - uses: vmware-labs/vmware-image-builder-action@0.4.11
```

Comment:
This change should not be here, right?

Reply:
Just a bad sync with main branch. Solved [aea8dc8](https://github.com/bitnami/charts/pull/13480/commits/aea8dc8eb9a80399a557b791cbe14c0f4012447b)